src/marker_cache.cpp

```diff
@@ -175,7 +175,15 @@ std::shared_ptr<mapnik::marker const> marker_cache::find(std::string const& uri,
             svg_path_adapter svg_path(stl_storage);
             svg_converter_type svg(svg_path, marker_path->attributes());
             svg_parser p(svg);
-            p.parse_from_string(known_svg_string);
+
+            if (!p.parse_from_string(known_svg_string))
+            {
+                for (auto const& msg : p.error_messages())
+                {
+                    std::cerr << "SVG PARSING ERROR:\"" << msg << "\"" << std::endl;
```

Comment:
@jakepruitt - yes, fixed in 3a4d57492fe68d3afe4ce2b02344dd5c0a78e1ed. I also made `marker_cache` to return `marker_null()` if case of any parsing/values errors. I think this is desired logic :  only accept SVG we can interpret correctly and fail otherwise + logging errors.